serokell-util.cabal

```diff
@@ -53,6 +52,7 @@ library
 
   build-depends:       base >= 4.9 && < 5
                      , aeson >= 1.0 && < 1.4
+                     , aeson-options == 0.0.0
```

Comment:
I find this bound too restrictive. Maybe we can just remove it...

Reply:
:ok_hand: